addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -73,8 +154,163 @@ def _stripe_make_request(self, endpoint, payload=None, method='POST', offline=Fa
+            'company[name]': self.company_id.name,
+            'individual[address][city]': self.company_id.city or '',
+            'individual[address][country]': self.company_id.country_id.code or '',
+            'individual[address][line1]': self.company_id.street or '',
+            'individual[address][line2]': self.company_id.street2 or '',
+            'individual[address][postal_code]': self.company_id.zip or '',
+            'individual[address][state]': self.company_id.state_id.name or '',
+            'individual[email]': self.company_id.email or '',
+            'business_profile[name]': self.company_id.name,
+        }
+
+    def _stripe_account_link_url(self, connected_account_id, menu_id):
+        """ Create an account link and return its URL.
+
+        An account link url is the beginning URL of Stripe Onboarding.
+        This URL is only valid once, and can only be used once.
+
+        :param str connected_account_id: The id of the connected account
+        :param int menu_id: The id of the menu to return to after onboarding.
+        :return: The account link URL
+        :rtype: str
+        """
+        return_url = urls.url_join(
+            self.company_id.get_base_url(),
+            StripeController._onboarding_return_url + f'{self.id}/{menu_id}'
+        )
+        refresh_url = urls.url_join(
+            self.company_id.get_base_url(),
+            StripeController._onboarding_refresh_url + f'{self.id}/{connected_account_id}/{menu_id}'
+        )
+        account_link = self._stripe_make_proxy_request('account_links', payload={
+            'account': connected_account_id,
+            'return_url': return_url,
+            'refresh_url': refresh_url,
+            'type': 'account_onboarding',
+        })
+        return account_link['url']
+
+    def _stripe_make_proxy_request(self, endpoint, payload=None, version=1):
+        """ Make a request to the Stripe proxy at the specified endpoint.
+
+        :param str endpoint: The proxy endpoint to be reached by the request
+        :param dict payload: The payload of the request
+        :param int version: The proxy version used
+        :return The JSON-formatted content of the response
+        :rtype: dict
+        :raise: ValidationError if an HTTP error occurs
+        """
+        proxy_payload = {
+            'jsonrpc': '2.0',
+            'id': uuid.uuid4().hex,
+            'method': 'call',
+            'params': {'payload': payload},
+        }
+        proxy_url = self.env['ir.config_parameter'].sudo().get_param(
+            'payment_stripe.fallback_proxy_url', FALLBACK_PROXY_URL
+        )
+        url = urls.url_join(proxy_url, f'{version}/{endpoint}')
+        try:
+            response = requests.post(url=url, json=proxy_payload, timeout=60)
+        except requests.exceptions.ConnectionError:
+            raise ValidationError(
+                "Stripe Proxy: " + _("Could not establish the connection to the API.")
```

Comment:
why translate half of the message?

Reply:
To have the same translation as here : https://github.com/odoo/odoo/blob/15.0/addons%2Fpayment_stripe%2Fmodels%2Fpayment_acquirer.py#L73 in stable.

But it may be useless/meaningless ? What is your advice ?

Reply:
Think languages with different alphabet (e.g. Arabic with right-to-left) mixing both English and translation will be quite odd. Just translate all the message.

Reply:
In payment modules, we generally don't translate the acquirer name (which is printed to give some context similar errors) because people would translate Authorize.net to Autoriser.nét, for example :roll_eyes: 

But, for Stripe proxy, I think it's fine if the name is translated as it's part of the error message.